mavros/launch/px4_config.yaml

```diff
@@ -96,6 +96,7 @@ setpoint_attitude:
     frame_id: "map"
     child_frame_id: "target_attitude"
     rate_limit: 50.0
+  thrust_scaling: 0.0
```

Comment:
Better move it before tf.

Reply:
And good idea to add comment that this parameter used only for RPYT.

Reply:
True! Will fix.